bluepysnap/circuit_validation.py

```diff
@@ -371,6 +409,10 @@ def _check_nodes_population(nodes_dict, config):
         nodes = _get_h5_data(h5f, 'nodes')
         if not nodes or len(nodes) == 0:
             return [fatal('No "nodes" in {}.'.format(nodes_file))]
+        populations_dict = nodes_dict.get('populations', dict())
```

Comment:
I'm surprised that lint didn't notice a plain `dict()`. In all recent PRs it requires it to be a literal.

Reply:
I will make another review on that since unpinning pylint would imply changes on files not changed in this PR.